free-programming-books-es.md

```diff
@@ -217,6 +217,7 @@
 ### JavaScript
 
 * [Guía de JavaScript 'Mozilla'](https://developer.mozilla.org/es/docs/Web/JavaScript/Guide) (HTML)
+* [El Tutorial de JavaScript Moderno](https://es.javascript.info/) (HTML)
```

Comment:
Hello, your build is failing because you need it to be in alphabetical order as well as remove the trailing slash.

Refer to the [CONTRIBUTING](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md) file

```suggestion
* [El Tutorial de JavaScript Moderno](https://es.javascript.info) (HTML)
* [Guía de JavaScript 'Mozilla'](https://developer.mozilla.org/es/docs/Web/JavaScript/Guide) (HTML)
```